client/src/app/components/nova/block/BlockTangleState.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from "classnames";
+import moment from "moment";
+import React, { useEffect, useState } from "react";
+import Tooltip from "../../Tooltip";
+import "../../stardust/block/BlockTangleState.scss";
```

Comment:
Should we copy the scss to a nova specific file (under nova folder) ? So that if we want to make it look different on nova we can ? @begonaalvarezd @VmMad

Reply:
Also make scss import be the last one